src/crewai/crew.py

```diff
@@ -69,21 +67,6 @@ def _deny_user_set_id(cls, v: Optional[UUID4]) -> None:
                 "may_not_set_field", "The 'id' field cannot be set by the user.", {}
             )
 
-    @classmethod
-    @field_validator("config", mode="before")
-    def check_config_type(
```

Comment:
Didn't find an use for this. 😬

Reply:
Oh keep it please haha It's coming soon :D

Reply:
Sure sure... I've just removed it because of `json.loads(v)` when `v` is a `Json` is failing, as `loads` expects a string. Can I simply return `v`, as it matches the return type?

Reply:
yup, I can fix it once I roll out the feature that will use this

Reply:
Cool, cool. I'll add a `#ignore` there then with a `TODO`.